src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,153 @@
 package core.basesyntax;
 
+//import java.util.HashMap;
+import java.util.Objects;
+import java.util.Random;
+
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public double EXPANSION_COEFFICIENT = 2;
+    public double LOAD_FACTOR = 0.75;
```

Comment:
Please rearrange fields of the class to be in the next order:
public static members
private static members
public non-static members
private non-static members

ref: Clean Code, Robert Martin